defender/poap/README.md

```diff
@@ -1,37 +1,103 @@
+- In the **Connect a Relayer** section, choose the Relayer that was used to deploy the contract
 
-```
-{
-  "address": "0xe3eFcCF966921Ad291fdC31Fa57F9044F105bc98",
-  "signature": "0xd1ea49a0576519030eb92182b9aa7a0e27b97816f1427a1cee00b11de3156c58095ae2ee7fab13cea7bd3db6aaacfcef596e995d49b0100026bc927f84b0c1311c",
-  "message": { "name": "Tim", "wallet": "0xe3eFcCF966921Ad291fdC31Fa57F9044F105bc98" } 
-}
+### Contract Was Deployed without Defender
+
+If the contract was not deployed with Defender, then the contract ownership or minting role will need to be assigned to the Defender Relay.
+
+- In your [Defender account](https://defender.openzeppelin.com/)
+- Click on the **Relay** link on the left side
+- In the **POAP Minter Relay** box, click on the **copy address** button
+- Use any tool to transfer ownership or minting roles to the POAP Minter Relay. Tools to transfer ownership include:
+  - Defender Admin
+  - Etherscan
+  - A simple script
+
+### Add Funds to the Relay
+
+- In your [Defender account](https://defender.openzeppelin.com/)
+- Click on the **Relay** link on the left side
+- Find the Relay that currently has ownership of the NFT contract, click on the **copy address** button
+- Transfer some funds that address to cover the NFT minting costs
+
+### Get the Autotask webhook URL
+
+- In your [Defender account](https://defender.openzeppelin.com/)
+- Click on the **Autotask** link on the left side
+- In the **POAP Minter** Autotask box, click on the **Copy** button to copy the Autotask Webhook URL to your clipboard
+- Save the Autotask URL to a local file
+
+## Typical Usage
+
+A participant will be required to sign a message using the private key associated with their account. In this template, the message only contains an address and a name. 
+
+### Signing the Message
+
+From the root of this repo:
+
+- Create/modify the `.env` file
+- Add `PRIVATE_KEY=<YOUR_PRIVATE_KEY>` to the file. Replacing the `<>` text with the private key of the account that will be receiving the POAP NFT
+- Run `yarn ts-node scripts/signPOAPMessage.ts --name <AttendeeName> --chainId 5` to generate a signature for Goerli Testnet (chain id 5)
+
+### Submitting the request
+The JSON that you created above will need to be POSTed to the Autotask webhook URL. This can be done with a simple script or with curl. Replace the URL with your Autotask webhook URL
+```js
+curl -X POST https://api.defender.openzeppelin.com/autotasks/xxxx/runs/webhook/xxxx \
+   -H 'Content-Type: application/json' \
+   -d '{
+  "address": "0xRECIPIENT_ADDRESS",
+  "signature": "0xVERY_LONG_SIGNATURE_STRING",
+  "message": { "name": "MyName", "wallet": "0xRECIPIENT_ADDRESS" }
+}'
 ```
 
-Have fun hacking! 
+## Notes
+It is best to add additional fields that can be used to verify a participant's attendance at an event. Moreover, some anti-spam measures should be in place to prevent users from submitting the same request repeatedly and minting NFTs until your Relay runs out of gas.
```

Comment:
```suggestion
It is best to add fields that can be used to verify a participant's attendance at an event. Moreover, some anti-spam measures should be in place to prevent users from submitting the same request repeatedly and minting NFTs until your Relay runs out of gas.
```